spellings/getFiles

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+bashPass="\033[32;1mPASSED -"
+bashInfo="\033[33;1mINFO -"
+bashFail="\033[31;1mFAILED -"
+bashEnd="\033[0m"
+
+# Check number of arguments
+files=""
+file=""
+excludeDirs=""
+excludeFiles=""
+includeExtensions=""
+
+# Make the assumption that Mac users will be using FreeBSD Get Opt
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    while [ $# -gt 0 ]; do
+    case "$1" in
+        -ed= | --exclude-dirs=* )
+            excludeDirs="-E $(echo "${1#*=}" | sed -E 's/,/ -E /g')"
+            shift
+            ;;
+        -ef= | --exclude-files=* )
+            excludeFiles="-E $(echo "${1#*=}" | sed -E 's/,/ -E /g')"
+            shift
+            ;;
+        -ie= | --include-extensions=* )
+            includeExtensions="-e $(echo "${1#*=}" | sed -E 's/,/ -e /g')"
+            shift
+            ;;
+        -h | --help )
+                echo -e "$bashInfo Find all .c and .h files with the Amazon copyright in them $bashEnd"
+                echo -e "$bashInfo It exports this to a bash array variable called \"files\" $bashEnd"
+                echo -e "$bashInfo This script can take in two optional arguments $bashEnd"
+                echo -e "$bashInfo -ef= | --exclude-files=:       A comma seperated list of files to exclude $bashEnd"
+                echo -e "$bashInfo -ed= | --exclude-dir=:         A comma seperated list of directories to exclude $bashEnd"
+                echo -e "$bashInfo -ie= | --include-extensions=:  Any additional exstensions to search for $bashEnd"
+                exit 0
+            ;;
+        *)
+            echo "Unknown option: $1"
+            exit 1
+            ;;
+        esac
+    done
+    # For FreeBSD (default MacOS shell) assume that you have fd
+    # By default only grab the c and h files. Only grab the files with amazon copyright
+    # Put all the files in an array. Uncomment the following line to see the command run.
+    # echo "fd -e c -e h "$excludeDirs" "$excludeFiles" "$includeExtensions" --exec grep -lie \"copyright (.*) 20[0-9]{2} amazon.com\" "
+    # Need this in a set +/- e so it doesn't return an error on files that fail the grep
+    set +e
+    files=$(fd -e c -e h $excludeDirs $excludeFiles $includeExtensions --exec grep -lie "copyright (.*) 20\d\d amazon.com")
+    set -e
+
+# Making the assumption that anybody else running this is a gnu getopt user
```

Comment:
Why can we assume that another user would be using `getopt`?

Reply:
Well it's an assumption that if you're using command line linux that you are using bash and using the default getopt for it. Where if you're not using it, well then guess this isn't a truly portable script

Reply:
Using fd-find now, removed this script entirely.